src/canadiantracker/http.py

```diff
@@ -55,10 +55,22 @@ async def products(request: Request):
     return _templates.TemplateResponse("index.html", {"request": request})
 
 
+def sanitize_sku(listing: ProductListingEntry):
+    if not listing.sku:
+        return listing.code
+
+    if "|" in listing.sku:
+        return listing.sku.split("|")[0]
```

Comment:
Not really important, but you could just do:

    return listing.sku.split("|")[0]

It will DTRT even if there is no `|` in the string.